### PR TITLE
Hotfix - UIowa Hours - Get today's date before submit

### PR DESCRIPTION
**Pull request by joewhitsitt**
Relates to https://github.com/uiowa/uiowa/issues/4651

get today's date with offset set as value before mousedown.

going to leave my window open over night to test an anonymous refresh tomorrow.

### CODE REVIEW

**Comment by joewhitsitt** [comment]
This worked as expected on that overnight browser window.

File: docroot/modules/custom/uiowa_hours/js/filter_form.js [hunk 1]
```diff
@@ -10,6 +10,8 @@
           $('.uiowa-hours-container', this).html('<p>Placeholder for ' + hoursConfig[0] + ' data</p>');
         }
         else {
+          let today = new Date().toLocaleDateString('en-CA', {timeZone: "America/Chicago"});
```
Comment (richardbporter) [inline docroot/modules/custom/uiowa_hours/js/filter_form.js]: Shouldn't we use `en-US` here instead of Canada?

Comment (richardbporter) [reply docroot/modules/custom/uiowa_hours/js/filter_form.js]: Nit: single quotes on `America/Chicago`. :)

Comment (joewhitsitt) [reply docroot/modules/custom/uiowa_hours/js/filter_form.js]: my understanding is that en-CA outputs the yyyy-mm-dd format (which the input expects) and en-US outputs mm/dd/yyyy https://www.w3schools.com/jsref/tryit.asp?filename=tryjsref_tolocalestring_date_all

Comment (richardbporter) [reply docroot/modules/custom/uiowa_hours/js/filter_form.js]: Oh I see. Thats annoying. Also found this https://www.drupal.org/project/drupal/issues/2875131.

**Review by richardbporter — APPROVED** [review]

**Pull request merged**